lib/net/nforceif/src/sys_arch.c

```diff
@@ -198,6 +198,12 @@ sys_mbox_trypost(struct sys_mbox **mb, void *msg)
   return ERR_OK;
 }
 
+err_t
+sys_mbox_trypost_fromisr(struct sys_mbox **mb, void *msg)
+{
+  return sys_mbox_trypost(mb, msg);
+}
+
```

Comment:
http://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=8fc20142f7b1b18f24fc48187d390187d718ce30 introduced the requirement to provide this function to address an issue that seems specific to FreeRTOS. I didn't spot any actual use of this / the function calling this, but the linker still demands it, so this just wraps `sys_mbox_trypost`.